mc-sema/cfgToLLVM/x86Instrs_INCDECNEG.cpp

```diff
@@ -228,34 +228,49 @@ static InstTransResult doDecR(InstPtr ip,  BasicBlock      *&b,
     return ContinueBlock;
 }
 
-template <int width>
+template <int width, int regWidth>
```

Comment:
I don't see how `width` and `regWidth` can ever be different for a decrement of a memory operand.